.github/workflows/build-master.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-static-page:
+    name: build
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: '1'
+    steps:
+      - name: print docker info
+        run: docker version && docker info
+      - uses: actions/checkout@v2
+      - name: build image
+        run: docker build --build-arg=ENABLE_ARCHIVES=true --target=current -t documentation:latest .
+      - name: copy static files locally
+        run: docker run -v ${PWD}:/output documentation:latest cp -r /usr/share/nginx/html /output/_site
```

Comment:
If we're not able to use the `docker build --output` option, possibly;

```
docker create --name tmp documentation:latest
docker cp tmp:/ /output/_site
```

We could change the `deploy-src` target to put everything in a `/_site` subdirectory if that's more convenient; https://github.com/docker/docker.github.io/blob/20267e58b96b1d41689e01eacd61172112017aa9/Dockerfile#L89

Reply:
Oh I just saw that we have all these stages in the Dockerfile. Maybe then it's easier to create a target stage with the AWS CLI installed (`FROM dockereng/awscli:1.16.156`), then we do not have to copy the files multiple times. Maybe saves some valuable GH actions build time.

Reply:
That image itself is private, so perhaps just repeat the steps in a stage, as all it does is an install of the cli;

```
RUN apk -Uuv add groff less python py-pip \
 && pip install awscli==${VERSION} \
 && apk --purge -v del py-pip
```

Tricky bit is that we'd have to pass the credentials to `docker build` as a build-arg, which is potentially dangerous, so if the original idea to use `docker build --output` works, that would be a cleaner option

Reply:
oh, actually, the image itself isn't private, but still likely that we could just install it inline in the dockerfile 🤔  (if the `--output` variant doesn't work out)

Reply:
Good idea. I've updated the `docker build` as shown in the `Dockerfile` and use the `--output` parameter.